docs/reference/cli.md

```diff
@@ -457,12 +440,9 @@ quickwit service run indexer
 
 *Examples*
 
-*Add and consume source*
+*Start an Indexer*
 ```bash
- 
-quickwit source add --id wikipedia_data_1 --index wikipedia --params '{"filepath":"wiki-articles_1.json"}' --type file
-quickwit service run indexer --indexes wikipedia
-
+quickwit service run indexer --config=./config/quickwit.yaml
```

Comment:
I prefer the examples to be more complete, and indexer without sources doesn't do anything

Reply:
good point, I will keep it.